Makefile

```diff
@@ -44,6 +44,7 @@ lint:
 .PHONY: mypy
 mypy:
 	python -m mypy --strict --install-types --non-interactive ick tests
+	@echo $$(grep -R '# type: ignore.*# FIX ME' ick tests | wc -l) FIX ME comments remain
```

Comment:
Appears to find .pyc files with the string as well; maybe add `--include='*.py'`

Reply:
I didn't want to list all the files, just have a visible reminder of the total.

Reply:
I added `--include`

Reply:
On second thought, the noisier longer list of files might be better.  Your choice.